alea/examples/gaussian_model.py

```diff
@@ -1,32 +1,58 @@
-from typing import Optional
+from typing import Dict, List, Optional
 
-from scipy import stats
 import numpy as np
+from scipy import stats
 
 from alea.model import StatisticalModel
 
 
 class GaussianModel(StatisticalModel):
+    """
+    A model of a gaussian measurement, where the model has parameters mu and sigma.
+    For illustration, we show how required nominal parameters can be added
+    to the init sigma is fixed in this example.
+
+    Args:
+        parameter_definition (dict or list, optional (default=None)):
+            definition of the parameters of the model
+    """
+
     def __init__(
             self,
-            parameter_definition: Optional[dict or list] = None,
+            parameter_definition: Optional[Dict or List] = None,
             **kwargs,
         ):
-        """
-        Initialise a model of a gaussian measurement (hatmu),
-        where the model has parameters mu and sigma
-        For illustration, we show how required nominal parameters can be added to the init
-        sigma is fixed in this example.
-        """
+        """Initialise a gaussian model."""
         if parameter_definition is None:
             parameter_definition = ["mu", "sigma"]
         super().__init__(parameter_definition=parameter_definition, **kwargs)
 
     def _ll(self, mu=None, sigma=None):
+        """
+        Log-likelihood of the model.
+
+        Args:
+            mu (float, optional (default=None)): mean of the gaussian,
+                if None, the nominal value is used
+            sigma (float, optional (default=None)): standard deviation of the gaussian,
+                if None, the nominal value is used
```

Comment:
should we warn that no nominal values are set here?

Reply:
We do catch it-- but it is nice to point out this stumbling block in the documentation, in particular since this is a tutorial model, that fails if you use it naively